portfolio/src/main/webapp/script.js

```diff
@@ -2,20 +2,32 @@
 // This function generates a random quote to display, also making sure that the previously displayed random image is not the same as the current random image.
 function randomizeQuote() {
 
+  const range= 4;    // This constant variable keeps track of the number of quotes I have. 
+
   // ImageNumber has the Image index of the quotes. This generates a number between [1-4]
-  var imageNumber = Math.floor(Math.random() * 4) + 1;
+  var imageNumber = Math.floor(Math.random() * range) + 1;
 
   // To check if an image exists already. The length of the list returned must have some element in it. This means that this is not the first time the button was clicked after page reload. 
   if($('#generated-quote-id').length>0)
   {
     var displayedImageNumber = $('#generated-quote-id').attr('src').substr(13,1);    // This gets the number(index) of the image that was displayed right before the button was clicked. 
 
-    while(imageNumber+"" === displayedImageNumber){        // Checks if the newly generated index is the same as the previously displayed image. as long as it is the same, it generates a different index
-      imageNumber = Math.floor(Math.random() * 4) + 1;        
+    // If the displayed picture and the currently generated image have the same index, 
+    if(imageNumber+"" === displayedImageNumber)
+    {
+      // And if the Number is the last value in the set of images, simply make sure to print the first picture of the bunch.
```

Comment:
nit: Move comment into if block (after if line) so it will match the else block.